include/nbl/ui/CWindowManagerX11.h

```diff
@@ -79,7 +156,17 @@ class CWindowManagerX11 : public IWindowManager
 		}
 	private:
 		Display* display;
+		X11 x11("X11");
+		Xinput xinput("Xinput");
+#ifdef _NBL_LINUX_X11_RANDR_
+    	Xrandr xrandr("Xrandr");
+#endif
+#ifdef _NBL_LINUX_X11_VIDMODE_
+    	Xxf86vm xxf86vm("Xxf86vm");
+#endif	
```

Comment:
dont even bother with the ifdef, remove the defines from the new code (and any place in the engine really)

Reply:
TODO: @sadiuk